.github/workflows/main.yml

```diff
@@ -1,5 +1,5 @@
 name: Build Extension
-on: [push, pull_request]
+on: pull_request
```

Comment:
Suggest we still run on pushes, only for the default branch. That way we have a constant indicator of whether the build is passing on the codebase.
```suggestion
on:
  pull_request:
  push:
    branches:
      - main
```

Reply:
Sure.